charts/node-analyzer/tests/collector_endpoint_region_test.yaml

```diff
@@ -0,0 +1,66 @@
+suite: Test benchmark runner collector_endpoint value for all regions
+templates:
+  - templates/configmap-benchmark-runner.yaml
+tests:
+  - it: Checking default value no region specified (us1)
+    asserts:
+      - isKind:
+          of: ConfigMap
+      - equal:
+          path: data.collector_endpoint
+          value: https://secure.sysdig.com
+  - it: Checking region 'us2'
+    set:
+      global: 
+        sysdig:
+          region: us2
+    asserts:
+      - isKind:
+          of: ConfigMap
+      - equal:
+          path: data.collector_endpoint
+          value: https://us2.app.sysdig.com
+  - it: Checking region 'us3'
+    set:
+      global: 
+        sysdig:
+          region: us3
+    asserts:
+      - isKind:
+          of: ConfigMap
+      - equal:
+          path: data.collector_endpoint
+          value: https://app.us3.sysdig.com
+  - it: Checking region 'us4'
+    set:
+      global: 
+        sysdig:
+          region: us4
+    asserts:
+      - isKind:
+          of: ConfigMap
+      - equal:
+          path: data.collector_endpoint
+          value: https://app.us4.sysdig.com
+  - it: Checking region 'eu1'
+    set:
+      global: 
+        sysdig:
+          region: eu1
+    asserts:
+      - isKind:
+          of: ConfigMap
+      - equal:
+          path: data.collector_endpoint
+          value: https://eu1.app.sysdig.com
+  - it: Checking region 'au1'
+    set:
+      global: 
+        sysdig:
+          region: au1
+    asserts:
+      - isKind:
+          of: ConfigMap
+      - equal:
+          path: data.collector_endpoint
+          value: https://app.au1.sysdig.com
```

Comment:
nit: adding a newline here would appease GitHub. 🙂

Reply:
got it